### ISSUE TITLE
Cannot edit pages with metadata set 

### ISSUE THREAD

**Issue by buarle** [TryGhost/Ghost#13940]
### Issue Summary

Environment
Version: 4.32.0
Environment: Production
Database: mysql (MariaDB 10.4)
Mail: SMTP

In general updating and creating pages is possible as long as the page has no metadata fields modified and you are not modifing them currently.
When there is metadata set, no modifiation is possilbe anymore.

Please advice how to proceed.

Thanks a lot,

Markus

### Steps to Reproduce

In general updating and creating pages is possible as long as the page has no metadata fields modified and you are not modifing them currently.
When there is metadata set, no modifiation is possilbe anymore.

### Ghost Version

4.32

### Node.js Version

The ohne from Latest docker image

### How did you install Ghost?

Docker-compose ghost:4

### Database type

MySQL 8

### Browser & OS version

Latest Edge

### Relevant log / error output

```shell
GUI Error Message
Internal server error, cannot edit page. Unable to update nested relation. [object Object]

Docker Logs Error Message
[2022-01-03 12:24:56] ERROR “PUT /ghost/api/canary/admin/pages/60264c84bda467000125a24a/” 500 119ms
Unable to update nested relation.
{“key”:“posts_meta”,“tableName”:“posts”,“method”:“setHasOne”}
Error ID:
2906e190-6c90-11ec-8030-a7ff1d07a2c6
Error Code:
UPDATE_RELATION

BookshelfRelationsError: Unable to update nested relation.
```

### Code of Conduct

- [X] I agree to be friendly and polite to people in this repository

**Comment by buarle** [comment]
FYI:
As a workaround you can delete the metadata within the database. After that editing is possible. Re-creating the metadata to re-importing it into the database does not work.

**Comment by matthanley** [comment]
Hey @buarle - I've been unable to reproduce this with arbitrary metadata. Could you provide more information on which fields you're updating that are causing the error?

**Comment by buarle** [comment]
Hi Matt,

in that stage I am unable to do any changes to the page. The condition is that the page needs to have metadata.

Also creating a fresh page and setting any kind of metadata prohibits me from publishing / saving anything.

I do not have any further logs than states unless somebody advises me how to gather them.

Best,
Buarle

**Comment by buarle** [comment]
Hi all,

this issue still exists in version 4.33.1 in all three blogs I'm hosting.
All within docker and all with separate databases mariadb:10.4

Please come bak to me and I will provide all requested information / logs /...

Best
Markus

**Comment by buarle** [comment]
Hi @matthanley,

i found a similar looking error with deleting users that was fixed with 3.40.3 according to that post:

https://gitanswer.com/trying-to-delete-a-member-through-ghost-admin-fails-with-2-critical-errors-javascript-ghost-770825647

How can we move on here?

Best
buarle
